packages/trader/src/AppV2/Components/Carousel/__tests__/carousel-header.spec.tsx

```diff
@@ -30,18 +34,28 @@ describe('CarouselHeader', () => {
         expect(info_icon).toBeInTheDocument();
 
         expect(mock_props.onNextClick).not.toBeCalled();
-        userEvent.click(info_icon);
+        await userEvent.click(info_icon);
         expect(mock_props.onNextClick).toBeCalled();
     });
 
-    it('should render correct icon for passed index. If user clicks on arrow icon, onPrevClick should be called', () => {
+    it('renders correct icon for passed index. If user clicks on arrow icon, onPrevClick should be called', async () => {
         render(<CarouselHeader {...mock_props} current_index={1} />);
 
         const arrow_icon = screen.getByText('LabelPairedArrowLeftMdRegularIcon');
         expect(arrow_icon).toBeInTheDocument();
 
         expect(mock_props.onPrevClick).not.toBeCalled();
-        userEvent.click(arrow_icon);
+        await userEvent.click(arrow_icon);
```

Comment:
Here and below all async-await were added automatically
<img width="798" alt="Screenshot 2024-11-13 at 5 43 43 PM" src="https://github.com/user-attachments/assets/4d66fd49-536e-4121-bc19-14b70b745713">